galactic-marketplace/utils.ts

```diff
@@ -0,0 +1,12 @@
+import { Order } from "@staratlas/galactic-marketplace";
+
+// FIXME: Should this be re-exported in galactic-marketplace it's defined in `utils.ts` but not exported again
```

Comment:
I'm not sure what you mean by "resolve". This is a copy of a convenience function from `@staratlas/galactic-marketplace/src/utils.ts` line #33 that is not publicly re-exported.

Reply:
A `FIXME` in code usually means, that something needs to be fixed. The comment is not clear to me. If this needs a fix, then it should be fixed. If not, then `FIXME` is the wrong annotation.

Reply:
OK, I changed it from `FIXME` to `TODO`. I could also strike the entire comment line. I just wanted to call to attention that this function exists in and is defined in `@staratlas/galactic-marketplace`. It would be up to whoever manages that package to re-export if it's "useful".

Reply:
I think, a simple comment explaining the situation would be good here. `FIXME` and `TODO` should always be avoided.